wpengine/faustjs
# MERL-1712: custom toolbar example

## Description
Added an example for customizing the toolbar.

## Testing

1. Setup a faust project using the custom-toolbar example
2. Login using the button on the main frontpage template to authenticate
3. Toolbar should display with the custom elements showing from CustomToolbar.tsx in the /plugins folder.

## Screenshots
Logged Out:
![Screenshot 2024-01-12 at 10 09 19 AM](https://github.com/wpengine/faustjs/assets/1815200/23b53a73-70ad-49e6-8840-e2243a161890)

Logged In:
![Screenshot 2024-01-12 at 10 09 28 AM](https://github.com/wpengine/faustjs/assets/1815200/4bd8ae08-a2fd-46e3-a073-25afd674250e)

Closes #1712
